### ISSUE TITLE
Add support for official libraries Unity

### ISSUE THREAD

**Issue by Eljeyna** [consulo/consulo-unity3d#201]
For example, Addressables, New Input System etc.
Maybe I'm doing something wrong, but external libraries works perfect...

**Comment by Eljeyna** [comment]
Addressables fixed with reimport in Unity and Consulo
Seems like bug with New Input System

**Comment by VISTALL** [comment]
Hello. Can you switch to nighlty (https://consulo.help/platform/update_channels/) and enable Experimental Features(Settings->Platform and Plugins -> Experimental Features) **New Unity Importer (with asmdef)** and Reimport Unity Project.

**Comment by Eljeyna** [comment]
Don't see this feature in nightly build
![image](https://user-images.githubusercontent.com/43903165/115657918-6dc82600-a340-11eb-9efd-11f6ac723f2f.png)

![image](https://user-images.githubusercontent.com/43903165/115658038-a8ca5980-a340-11eb-89cd-9f305cddb905.png)

**Comment by VISTALL** [comment]
About -> Check for Updates. You changed channel, but not updated to last builds

**Comment by Eljeyna** [comment]
Sorry for my mistakes. After some retries i am break Consulo in Windows and on Linux Importer doesn't change anything

**Comment by VISTALL** [comment]
You can show me errors - and I will say what you need

**Comment by Eljeyna** [comment]
> When New Input System create a settings file, some fields "doesn't exists". With that class not visible in other scripts (but it works in Unity)
> 
> UPD: no problem with reimport in Consulo and preprocessor variables
> 
> NewInputSystem.cs
> ![image](https://user-images.githubusercontent.com/43903165/115499460-270ef900-a278-11eb-92d5-6ddb28634b05.png)
> 
> Player.cs
> ![image](https://user-images.githubusercontent.com/43903165/115499698-8836cc80-a278-11eb-9037-371bac505862.png)

Error: "is not resolved" (red fields), no others (class found in Consulo, but marked "red" in struct and other classes)

**Comment by VISTALL** [comment]
It's dependency problem. If new Unity import not fix - I will test it at weekends

**Comment by VISTALL** [comment]
Hmm... Can you say - where class NewInputSystem? I'm idk where. If it's from you project, can you say file path, also class declaration (if it inside #if)

**Comment by Eljeyna** [comment]
Filepath is not necessary (but in my main project: Assets -> Scripts -> InputSystem).
It's simple to get it. Install Input System 1.02 from Package Manager. Enable New Input System (Unity will ask about it) or Edit -> Project Settings -> Player -> Active Input Handling (Input System Package (New)). Then Edit -> Input System Package -> Create Settings Asset. In Assets folder Create -> Input Actions. Double click on it (or Edit Asset). In left corner create new scheme (for example, keyboard), add new action (for example, space button in search). Save asset. Close settings. In Inspector click "Generate C# class" and Apply. Automatically created required file.

![image](https://user-images.githubusercontent.com/43903165/115947708-1ce83700-a4d2-11eb-850c-841f66abe025.png)

![image](https://user-images.githubusercontent.com/43903165/115947727-3db08c80-a4d2-11eb-8319-e289c9651a69.png)

And my folder if it's needed
![image](https://user-images.githubusercontent.com/43903165/115947826-10b0a980-a4d3-11eb-955f-a5803b58e56a.png)

**Comment by VISTALL** [comment]
Hm. Tested and looks like I found bug. 

Test it. Just remove **@** from type declaration. Like **class @NewInputSystem** -> **class NewInputSystem**.

Looks like bug inside C# analyzing

**Comment by Eljeyna** [comment]
Full refactoring helps (strange, it's didn't works before).
Пользуясь случаем, сорян за английский :D (я его использовал для общего понимания), но я его плохо знаю.
Thanks for excellent IDE (love it)!
Close this thread? There is one more bug (open the new one?)

**Comment by VISTALL** [comment]
Nope. If bug not releated to this issue - create new issue. This issue related to **@** bug in C# plugin. I will fix it in few days.

Нет. Если баг не касается это задачи - создавайте новую задачку. Эта задача касается **@** бага внутри C# плагина. Я поправлю в течении пару ней.

Thanks
